docs/runbooks/Container-README.md

```diff
@@ -2,340 +2,265 @@
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - "${POSTGRES_PORT}:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  rust-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: runtime
+    platform: ${PLATFORM}
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_HOST: ${POSTGRES_HOST}
+      POSTGRES_PORT: ${POSTGRES_PORT}
+      DATABASE_URL: ${POSTGRES_HOST}://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+      SERVICE_PORT: ${SERVICE_PORT}
+    ports:
+      - "${SERVIE_PORT}:4000"
+    depends_on:
+      - postgres
+volumes:
+  postgres_data:
+```
 
-From the project root, run the following command to build the Docker images for both the Rust application and PostgreSQL (if running locally).
+---
 
-```bash
-docker-compose --env-file .env.local build
-```
+## Building and Running the Application
 
-or for remote PostgreSQL:
+### **1. Build the Rust Backend Image**
 
 ```bash
-docker-compose --env-file .env.remote-db build
+docker buildx build --platform linux/amd64,linux/arm64 -t rust-backend .
 ```
 
-### **Step 2: Start the application**
-
-Once the build is complete, start the containers using Docker Compose. This will start the PostgreSQL database (if local) and the Rust application.
+This builds the image for both `amd64` and `arm64` architectures. Use the `--platform` flag to build for a specific architecture.
 
-For local PostgreSQL:
+### **2. Build and Run with Docker Compose**
 
+#### For Local PostgreSQL:
 ```bash
-docker-compose --env-file .env.local up
+docker-compose --env-file .env.local up --build
 ```
 
-For remote PostgreSQL:
-
+#### For Remote PostgreSQL:
 ```bash
-docker-compose --env-file .env.remote-db up
+docker-compose --env-file .env.remote-db up --build
 ```
 
-The Rust web API should now be running on `http://localhost:8080` and PostgreSQL should be available on port `5432` (or remotely if using the `.env.remote-db` setup).
+The web API will be accessible at `http://localhost:<SERVICE_PORT>` .
 
-## Using the Database Utilities
+---
 
-### **Rebuild the database**
+## Database Utilities
 
-To rebuild the database (create a new database, user, and schema):
+### **Rebuild the Database**
 
 ```bash
 docker-compose run rust-app rebuild-db
 ```
 
-This will use the default settings from your environment variables (`POSTGRES_DB`, `POSTGRES_USER`, and `POSTGRES_SCHEMA`).
-
-### **Seed the database**
-
-To seed the database with test data:
+### **Seed the Database**
 
 ```bash
 docker-compose run rust-app seed-db
 ```
 
-This runs a Rust service that seeds the PostgreSQL database with predefined data.
-
 ### **Convert DBML to SQL**
 
-If you have a DBML file (`schema.dbml`), you can convert it into an SQL schema:
-
+If you have a DBML file (`schema.dbml`), convert it to SQL:
```

Comment:
This part isn't really needed since a developer is responsible for converting the DBML diagram to a .sql file as part of their commit if they change the DB's table structures.